roofit/roofitcore/inc/RooFit/UniqueId.h

```diff
@@ -61,10 +61,17 @@ struct UniqueId {
    bool operator==(UniqueId const &other) const { return _val == other._val; }
    bool operator<(UniqueId const &other) const { return _val < other._val; }
 
-   static const UniqueId nullid;           ///< An ID that is less than the ID of any object (similar to nullptr).
+   /// Get an ID that is less than the ID of any object (similar to nullptr).
+   static UniqueId const& nullid() {
+      static const UniqueId nid{nullval};
+      return nid;
+   }
+
```

Comment:
Would not be better to use as Value_t ULong64_t  or uint64_t, since unsigned long is not portable and on some platform is a 32 bits integer ?

Reply:
Ok, it's not really necessary because this doesn't take part in IO but probably better to use `uint64_t` to avoid any surprises.